GuildedRoseLLCSpec/Fakes/FakeItemRepository.swift

```diff
@@ -0,0 +1,16 @@
+import GuildedRoseLLC
+
+public class FakeItemRepository: ItemRepository {
```

Comment:
What is the reason this class is made public? I would expect that the tests are the only thing using the `FakeItemRepository`.